worldedit-core/src/main/java/com/sk89q/worldedit/command/GenerationCommands.java

```diff
@@ -122,67 +122,92 @@ public void cyl(Player player, LocalSession session, EditSession editSession, Pa
     }
 
     @Command(
-        aliases = { "/hsphere" },
-        usage = "<block> <radius>[,<radius>,<radius>] [raised?]",
-        desc = "Generates a hollow sphere.",
-        help =
-            "Generates a hollow sphere.\n" +
-            "By specifying 3 radii, separated by commas,\n" +
-            "you can generate an ellipsoid. The order of the ellipsoid radii\n" +
-            "is north/south, up/down, east/west.",
-        min = 2,
-        max = 3
+            aliases = { "/hemisphere" },
```

Comment:
No, the indentation is incorrect, for all the annotations and some of the `case` statements. I know it's a minor thing but personally I'd prefer the whole project to stick to one style as much as possible 👍 

If you need to see what's wrong, take a look at the diff for the commit and notice the whitespace changes.